apps/system/utils/Kconfig

```diff
@@ -113,13 +113,15 @@ endif #!DISABLE_ENVIRON
 config ENABLE_FREE
 	bool "free"
 	default y
+	select MEMINFO if BUILD_PROTECTED
 	---help---
 		Show information about memory status
 
 config ENABLE_HEAPINFO
 	bool "heapinfo"
 	default y
 	depends on DEBUG_MM_HEAPINFO && FS_PROCFS
```

Comment:
Even we use meminfo driver, do we still need FS_PROCFS ?

Reply:
Yes. It still use procfs for task information like pid, stack, name